test/index.js

```diff
@@ -27,15 +27,15 @@ describe('Errors', () => {
                 return done();
             }
 
-            expect(result).to.equal({ data: { name: 'Error', stack: 'Some\nstack', message: 'foo' } });
+            expect(result.data.stringifyable).to.equal({ name: 'Error', stack: 'Some\nstack', message: 'foo' });
         });
 
         const err = new Error('foo');
         err.stack = 'Some\nstack';
         stream.end({ data: err });
     });
 
-    it('converts boom wrapped errors to object literals', (done) => {
+    it('add stringifyable property to boom wrapped error objects', (done) => {
```

Comment:
Hmm maybe that won't work, maybe a different test with:

Boom.internal('message', { some: 'other', important: 'props' })